{{cookiecutter.repostory_name}}/bin/backup-db.sh

```diff
@@ -11,15 +11,29 @@ if [ "$(basename "$0")" == 'bin' ]; then
 fi
 
 . .env
-
-target="db_dump_$(date +%Y-%m-%d_%H%M%S).sql.gz"
+mkdir -p .backups
+TARGET=".backups/db_dump_$(date +%Y-%m-%d_%H%M%S).sql.gz"
 
 if [[ "$DATABASE_URL" =~ "@db:" ]]; then
   DOCKER_NETWORK={{cookiecutter.repostory_name}}_default
 else
   DOCKER_NETWORK=host
 fi
 
-docker run --rm --network $DOCKER_NETWORK postgres:9.6-alpine pg_dump -c "$DATABASE_URL" | gzip > "$target"
+docker run --rm --network $DOCKER_NETWORK postgres:9.6-alpine pg_dump -c "$DATABASE_URL" | gzip > "$TARGET"
+echo "$TARGET"
+
+if [ -n "$EMAIL_HOST" ]; then
+  bin/backup-db-to-email.sh "$TARGET" || true
```

Comment:
I wouldn't want script to stop here and not proceeed to B2 backup because it couldn't sent an email. Once we add sentry integration, i can rework this.

Reply:
If we use sentry, it works for me. @ppolewicz ?